src/clients/lan_auto.rb

```diff
@@ -189,28 +185,18 @@ def ToAY(settings)
         "keep_install_network",
         Ops.get_boolean(settings, "keep_install_network", true)
       )
-      if Ops.greater_than(Builtins.size(modules), 0)
-        Ops.set(ret, "modules", modules)
-      end
+      Ops.set(ret, "modules", modules) if Ops.greater_than(Builtins.size(modules), 0)
       Ops.set(ret, "dns", dns) if Ops.greater_than(Builtins.size(dns), 0)
-      if Ops.greater_than(Builtins.size(dhcpopts), 0)
-        Ops.set(ret, "dhcp_options", dhcpopts)
-      end
+      Ops.set(ret, "dhcp_options", dhcpopts) if Ops.greater_than(Builtins.size(dhcpopts), 0)
       if Ops.greater_than(
         Builtins.size(Ops.get_map(settings, "routing", {})),
         0
       )
         Ops.set(ret, "routing", Ops.get_map(settings, "routing", {}))
       end
-      if Ops.greater_than(Builtins.size(interfaces), 0)
-        Ops.set(ret, "interfaces", interfaces)
-      end
-      if Ops.greater_than(Builtins.size(s390_devices), 0)
-        Ops.set(ret, "s390-devices", s390_devices)
-      end
-      if Ops.greater_than(Builtins.size(net_udev), 0)
-        Ops.set(ret, "net-udev", net_udev)
-      end
+      Ops.set(ret, "interfaces", interfaces) if Ops.greater_than(Builtins.size(interfaces), 0)
+      Ops.set(ret, "s390-devices", s390_devices) if Ops.greater_than(Builtins.size(s390_devices), 0)
+      Ops.set(ret, "net-udev", net_udev) if Ops.greater_than(Builtins.size(net_udev), 0)
```

Comment:
You may want to use the eager mode of the Zombie Killer for this, https://github.com/yast/zombie-killer/pull/9 , to translate `Ops.greater_than(Builtins.size(net_udev), 0)` into `!net_udev.empty?`

Reply:
well, goal if this pr is to adopt new rubocop. Feel free to open another one with zombie killer improvements